model/file_info.go

```diff
@@ -112,13 +111,12 @@ func (o *FileInfo) IsImage() bool {
 	return strings.HasPrefix(o.MimeType, "image")
 }
 
-func GetInfoForBytes(name string, data []byte) (*FileInfo, *AppError) {
+func GetInfoForBytes(name string, file io.Reader) (*FileInfo, *AppError) {
 	info := &FileInfo{
 		Name: name,
-		Size: int64(len(data)),
+		//Size: int64(len(data)),
```

Comment:
See comments above. If this is truly not needed anymore, we should remove the comment.